app/index.js

```diff
@@ -458,12 +458,20 @@ JhipsterGenerator.prototype.app = function app() {
     this.angularAppName = _.camelize(_.slugify(this.baseName)) + 'App';
     this.camelizedBaseName = _.camelize(this.baseName);
     this.slugifiedBaseName = _.slugify(this.baseName);
+    this.capitalizedBaseName = _.capitalize(this.baseName);
```

Comment:
Not sure we should add a new variable here. Besides, it always provides issues like name clashes, etc